tests/unit/models/transactions/test_did_set.py

```diff
@@ -62,3 +62,46 @@ def test_empty(self):
                 error.exception.args[0],
                 "{'did_set': 'Must have one of `did_document`, `data`, and `uri`.'}",
             )
+
+    def test_all_empty_fields(self):
+        with self.assertRaises(XRPLModelException):
+            DIDSet(account=_ACCOUNT, data="", did_document="", uri="")
+
+    def test_remove_multiple_fields(self):
```

Comment:
IMO each transaction should be a separate test. Easier to run.

Reply:
I'd like to remove consecutive fields in the DID object, in order to demonstrate that users cannot leave behind an empty DID object.

I need to perform consecutive removals on the same object, it cannot be demonstrated with a new DID ledger object

Reply:
That's not a library test, that's a rippled test. And even if it were a library test, it should happen in an integration test, because it won't happen in a unit test (the unit test doesn't submit transactions and therefore doesn't create/modify/delete ledger objects). All you're doing here is creating the models, not submitting anything.

Reply:
okay, I've created an integration test for the same